picard/ui/itemviews.py

```diff
@@ -284,6 +285,10 @@ def remove(self, objects):
         else:
             self.update_current_view()
 
+    def set_sorting(self, sort=True):
+        self.views[0].setSortingEnabled(sort)
+        self.views[1].setSortingEnabled(sort)
```

Comment:
What are `0` and `1` here?

Reply:
Left (clusters) and right (albums) panels

Reply:
We should name 2 constants for those, and use them instead of numeric values.

Reply:
How about the following?
```
class Panels(Enum):
    CLUSTERS = 0
    ALBUMS = 1

[...]

def set_sorting(self, sort=True):
    self.views[Panels.CLUSTERS].setSortingEnabled(sort)
    self.views[Panels.ALBUMS].setSortingEnabled(sort)
```